monitoring/repo_updater.go

```diff
@@ -147,30 +148,30 @@ func RepoUpdater() *Container {
 							Description:       "repositories managed by the scheduler",
 							Query:             `src_repoupdater_sched_known_repos`,
 							DataMayNotExist:   true,
-							Warning:           Alert{GreaterOrEqual: 1000, For: 5 * time.Minute}, // NOTE: There is really no point to have such warning
+							Warning:           Alert{LessOrEqual: 1, For: 10 * time.Minute},
 							PanelOptions:      PanelOptions().Unit(Number),
 							Owner:             ObservableOwnerCloud,
-							PossibleSolutions: "none",
+							PossibleSolutions: "Check repo-updater logs. This is expected to fire if there are no user added code hosts",
 						},
 						Observable{
 							Name:              "sched_update_queue_length",
 							Description:       "repositories queued for update",
 							Query:             `src_repoupdater_sched_update_queue_length`,
 							DataMayNotExist:   true,
-							Warning:           Alert{GreaterOrEqual: 1000, For: 5 * time.Minute}, // NOTE: There is really no point to have such warning
+							Warning:           Alert{LessOrEqual: 1, For: 8 * time.Hour},
 							PanelOptions:      PanelOptions().Unit(Number),
 							Owner:             ObservableOwnerCloud,
-							PossibleSolutions: "none",
+							PossibleSolutions: "Check repo-updater logs. This is expected to fire if there are no user added code hosts",
 						},
 						Observable{
 							Name:              "sched_loops",
 							Description:       "scheduler loops",
 							Query:             `rate(src_repoupdater_sched_loops[1m])`,
 							DataMayNotExist:   true,
-							Warning:           Alert{GreaterOrEqual: 10, For: 5 * time.Minute}, // NOTE: There is really no point to have such warning
+							Warning:           Alert{LessOrEqual: 1, For: 8 * time.Hour},
 							PanelOptions:      PanelOptions().Unit(Number),
 							Owner:             ObservableOwnerCloud,
-							PossibleSolutions: "none",
+							PossibleSolutions: "Check repo-updater logs. This is expected to fire if there are no user added code hosts",
```

Comment:
```suggestion
							PossibleSolutions: "Check repo-updater logs for TODO. This is expected to fire if there are no user added code hosts",
```

similar for the other solutions that say check logs